src/v2/Apps/Auction/Components/LotInfo.tsx

```diff
@@ -16,28 +17,32 @@ export const LotInfo: React.FC<Props> = ({ artwork, saleArtwork }) => {
     counts: { bidderPositions: bidCount },
   } = saleArtwork
   return (
-    <Flex py={4}>
+    <Flex id="our-parent-flex-box" py={4}>
```

Comment:
Why add an `id` here?

Reply:
Oh whoops that was so we could see it in the UI. thanks for catching that. will remove